digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/csvDataImporter/MassTransitStopCsvImporter.scala

```diff
@@ -426,7 +423,7 @@ class PositionUpdater (vvhClientImpl: VVHClient, roadLinkServiceImpl: RoadLinkSe
     val lat = getPropertyValue(properties, "maastokoordinaatti_y").asInstanceOf[BigDecimal].toDouble
     val externalId = getPropertyValue(properties, "external_id").toString.toInt
 
-    val roadLink = roadLinkService.enrichRoadLinksFromVVH(getNearestRoadLink(lon, lat, user, roadTypeLimitations))
```

Comment:
Why delete the enrich? We should update the roadlink we get from getNearestRoadLink method with some of our values in database right?

Reply:
The enrich is done on `def getClosestRoadlinkForCarTrafficFromVVH` on RoadLinkService.scala